src/main/java/net/mcreator/ui/modgui/codeviewer/ModElementCodeViewer.java

```diff
@@ -177,10 +177,14 @@ private synchronized void reload() {
 					setBackground(new Color(0x8D5C5C));
 				}
 
-				if (getTabCount() == 0)
+				int noFilesTab = indexOfTab(L10N.t("mod_element_code_viewer.no_files"));

```

Comment:
Not sure how this other change is related to the removal of usage tips.

More so as this change introduces a bug where in some cases there is no tab at all:

![image](https://github.com/user-attachments/assets/0552581e-2156-4631-807c-63fab744b641)

Please split off this into another PR and make sure it works as it should with sufficient testing. It took me 15 seconds to find this bug.